crates/e2e/src/setup/mod.rs

```diff
@@ -69,6 +69,18 @@ const DEFAULT_FILTERS: [&str; 9] = [
     "orderbook::api::request_summary=off",
 ];
 
+fn take_filters<T>(custom_filters: Option<impl IntoIterator<Item = T>>) -> Vec<String>
```

Comment:
Sorry, I just needed some function name for my example. 😅 
In this case sth like `with_default_filters` or so would make sense to me.